easybuild/easyconfigs/m/MFiX/MFiX-24.3-intel-2023b.eb

```diff
@@ -0,0 +1,48 @@
+# Author: Ehsan Moravveji (VSC, KU Leuven)
+
+easyblock = 'CMakeMake'
+
+name = 'MFiX'
+version = '24.3'
+
+homepage = 'https://mfix.netl.doe.gov/products/mfix/'
+description = """
+The National Energy Technology Laboratory’s (NETL’s) computational fluid dynamics (CFD) code, 
+MFiX-Multiphase Flow with Interphase eXchanges - is central to the laboratory’s multiphase flow reactor 
+modeling efforts. This open-source software has over three decades of development history and more than 7,000 
+registered users worldwide.  MFiX has become the standard for comparing, implementing, and evaluating multiphase 
+flow constitutive models and has been applied to an extremely diverse range of multiphase flows applications. 
+The successes achieved in modeling complex multiphase flow systems have led to new and improved key attributes 
+such as drag, polydispersity, attrition, and agglomeration models, among other significant advances.
+"""
+
+toolchain = {'name': 'intel', 'version': '2023b'}
+toolchainopts = {
+    'usempi': True, 
+    'pic': True, 
+    'extra_fflags': '-fallow-argument-mismatch'
+}
+
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['d6313d02b4631a6b2eef944c158bb7b1a2eecfe7bcee9f533ce71571f1acffe8']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('binutils', '2.40'),
+]
+
+configopts = '-DENABLE_MPI=1'
+
+separate_build_dir = False
+
+test_cmd = ' && '.join([
+    'cd tutorials/dem/mixer_3d',
+    '%(builddir)s/%(namelower)s-%(version)s/mfixsolver_dmp -f mixer_dem_3d.mfx',
+])
+
+sanity_check_paths = {
+    'files': ['mfixsolver_dmp'],
```

Comment:
This seems a bit strange, it doesn't seem to install to a `bin` directory? That would mean that the final binary is not available via `PATH` without some tweaking